source/modules/collection-api/sale/actions/mark_fraudulent.md.erb

```diff
@@ -0,0 +1,20 @@
+### Mark Sale Fraudulent
+```shell
+curl -XPOST https://collection.inspcdn.net/sale/mark_fraudulent \
+    -d '{
+        "sale_id":           12345,
+        "fraud_signal_type": "chargeback"
+    }'
+```
+Additionally, there is a `/sales/mark_fraudulent` endpoint that you should use to update Inspetor when a sale has been rejected or refunded due to suspicion of fraud, or if a sale has resuled in a chargeback.
+
+If a sale is deemed fraudulent (either via your existing antifraud solution, manual analysis, acquirer rejection, or even if you receive a chargeback), it is extremely important that you relay this information to Inspetor. This is perhaps _the most valuable piece of information_ you can send to us; all of our fraud prediction models are trained on this label. 
+
+The `/sale/mark_fraudulent` endpoint supports the `POST` operation and requires the following arguments:
+
+Property              | Required? | Type   | Description
+--------              | --------- | ----   | -----------
+**sale_id**           | Yes       | String | The unique identifier for the sale within your platform
+**fraud_signal_type** | Yes       | String | The mechanism by which you identified the sale as fraudulent. Current accepted values are 'acquirer', other antifraud services such as **'horus'**, **'fraudnet'**, and **'sift'**, your company name (**'ingresse'**), or simply **'chargeback'**.
+
+Note: it is expected that when marking a sale fraudulent, one would also make a corresponding call to the `/sales` endpoint to update a sale with a `rejected` status (unless the sale was marked fraudulent only after a chargeback was received). 
```

Comment:
Would it be a bad idea if we automatically updated the status of the sale based on the `fraud_signal_type`, so the client wouldn't have to call the `/sales` endpoint?  We could do something like, if it's chargeback don't update the sale, else set it to rejected.

Reply:
It just feels weird to have an optional parameter that is actually invalid given the value of another parameter (e.g. status==`approved`, then `fraud_signal_type` is not a valid param)